perception/object_recognition/detection/roi_cluster_fusion/CMakeLists.txt

```diff
@@ -2,27 +2,30 @@ cmake_minimum_required(VERSION 3.5)
 project(roi_cluster_fusion)
 
 if(NOT CMAKE_CXX_STANDARD)
-	set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 14)
 endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
 ament_auto_find_build_dependencies()
-find_package(OpenCV REQUIRED)
+find_package(OpenCV 4 REQUIRED)
```

Comment:
This can be built with opencv version 3, so it's not necessary to specify version.

Reply:
fixed at
5c6743e